slice/Glacier2/Metrics.ice

```diff
@@ -17,7 +17,7 @@
 
 #include <Ice/Metrics.ice>
 
-[["java:package:com.zeroc"]]
```

Comment:
So the fix is to use package com.zeroc.Glacier2.IceMX for the Glacier2-related MX code, as opposed to the current "split-package" com.zeroc.IceMX for all MX-related code?

Is com.zeroc.IceMX + com.zeroc.Glacier2.IceMX consistent?

Should it be instead com.zeroc.Ice.IceMX + com.zeroc.Glacier2.IceMX?

Or com.zeroc.Ice.MX + com.zeroc.Glacier2.MX? This would require Slice-compiler updates.

Reply:
> So the fix is to use package com.zeroc.Glacier2.IceMX for the Glacier2-related MX code, as opposed to the current "split-package" com.zeroc.IceMX for all MX-related code?

Yes, the issue is `com.zeroc.IceMX` package was split across Ice, Glacier2 and IceStorm jars

> Should it be instead com.zeroc.Ice.IceMX + com.zeroc.Glacier2.IceMX?

This would be indeed more consistent naming.

Reply:
Fixed as suggested